### PR TITLE
Add test 'Quick edit and bulk actions carriers'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add test 'Quick edit and bulk actions carriers'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/09_shipping/01_carriers/03_QuickEditAndBulkActions.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21573)

### CODE REVIEW

**Review by SimonGrn — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/09_shipping/01_carriers/03_QuickEditAndBulkActions.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+const carriersPage = require('@pages/BO/shipping/carriers');
+const addCarrierPage = require('@pages/BO/shipping/carriers/add');
+
+// Import data
+const CarrierFaker = require('@data/faker/carrier');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_shipping_carriers_quickEditAndBulkActions';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/09_shipping/01_carriers/03_QuickEditAndBulkActions.js:21]:
```suggestion
const baseContext = 'functional_BO_shipping_carriers_QuickEditAndBulkActions';
```

Comment (SimonGrn) [reply tests/UI/campaigns/functional/BO/09_shipping/01_carriers/03_QuickEditAndBulkActions.js:21]: Not sure that's how we write them ?

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/09_shipping/01_carriers/03_QuickEditAndBulkActions.js:21]: Yes, it is a small error, I will change the name of the file :+1: 
The first letter must be lowercase

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/09_shipping/01_carriers/03_QuickEditAndBulkActions.js:21]: Done thx :)

**Review by SimonGrn — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by SimonGrn — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**